ci/patch-fluent-bit-version.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+# Patches the Dockerfiles with the Fluent Bit version retrieved from the build tag.
+# It assumes that the Fluent Bit version is the first part of the argument passed into the function.
+# For example, calling `patch_fluent_bit_version "1.2.3-sumo-x.y.z"` will set Fluent Bit version to 1.2.3.
+function patch_fluent_bit_version() {
+    # Remove everything after first `-sumo` (including), e.g. `1.2.3-sumo-1-rc.1` => `1.2.3`
+    fb_version=${1%%-*}
+    # Check that Fluent Bit version is in correct format (SemVer)
+    [[ "$fb_version" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] || exit 10
```

Comment:
`[digits].[digits].[digits]` is not strictly semVer

Reply:
I mean it is, but semver is not always only `[digits].[digits].[digits]` :sweat_smile: